src/uvw/fs_event.hpp

```diff
@@ -134,8 +134,8 @@ class FsEventHandle final: public Handle<FsEventHandle, uv_fs_event_t> {
      * @param path The file or directory to be monitored.
      * @param watch Additional flag to control the behavior.
      */
-    void start(std::string path, Watch watch) {
-        start(std::move(path), Flags<Watch>{watch});
+    void start(std::string path, Event watch) {
```

Comment:
I would change also the name of the variable. `flag` seems more appropriate, for we are going to pack it in a `Flags` class and invoke the other overload around.